### ISSUE TITLE
Include MarkerClusterer Script/Class in 

### ISSUE THREAD

**Issue by phihochzwei** [angular/components#24130]
### Feature Description

Currently, when using @angular/google-maps you get access to the <map-marker-clusterer>-component. However, you have to manually add the script to load MarkerClusterPlus in your index.html to make it actually work, which is quite weird to understand. Therefore it would be great, also in terms of lazy loading, to include this class into the package to remove the necessity to add it manually

### Use Case

Basically any use case where you need the clustering :)

**Comment by crisbeto** [comment]
Duplicate of https://github.com/angular/components/issues/17037.

**Comment by phihochzwei** [comment]
Out of curiosity: how is this a duplicate? It has nothing to do with the Maps API loading itself but the loading of a different script 🧐

**Comment by crisbeto** [comment]
The clusterer is part of the maps API and if we add automatic script loading for the Google Maps API, we'll likely do the same for the clusterer.
